### ISSUE TITLE
image.id versus image.imageId

### ISSUE THREAD

**Issue by mermich** [smartsheet-platform/smartsheet-csharp-sdk#119]
Image object inside cells doesn't have the same structure as user profile image. 

While reading a cell containning an image the object is correctly parsed, yet on the user profile image the id property is empty.

Cell response from reading a sheet ( https://api.smartsheet.com/2.0/sheets/999999999?level=2 ): 
```
{
	"columnId": 761346099832708,
	"value": "Annotation 2020-03-26 082946.png",
	"displayValue": "Annotation 2020-03-26 082946.png",
	"formula": "=SYS_CELLIMAGE(\"Annotation 2020-03-26 082946.png\",\"gEQqGCr7KyzXcVoWi7J-tC\",168,878,\"Annotation 2020-03-26 082946.png\")",
	"image": {
		"id": "gEQqGCr7KyzXcVoWi7J-tC",
		"height": 168,
		"width": 878,
		"altText": "Annotation 2020-03-26 082946.png"
	}
}
```

User profile reponse (https://api.smartsheet.com/2.0/users/me)
```
"profileImage": {
	"imageId": "u!1!2v1Yf0456Q!U1PZ3znPe_w!DQqovAB1kle",
	"height": 1050,
	"width": 1050
}
```

Both objets refers to the same c# class Smartsheet.Api.Models.Image 
```
/// <summary>
/// Get Image Id.
/// </summary>
/// <returns> the Id </returns>
public string ImageId
{
    get { return imageId; }
    set { imageId = value; }
}
```

I can imagine 2 solutions: introduce a new class for user profile image (and avoid regressions for other languages) or  modify the curl response of userprofile.

**Comment by irwando** [comment]
Copied to internal backlog

**Comment by timwellswa** [comment]
Fixed in build 2.101.0
